mex/scripts/purge.py

```diff
@@ -0,0 +1,27 @@
+import click
```

Comment:
i'd put this outside the mex folder entirely. so into `mex-backend/scripts/purge.py` that way it's clear, this is not production code, wont count towards coverage and wont be packaged in the wheel distro